Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java

```diff
@@ -219,9 +217,10 @@ private static FileOutputStream updateCertEntry(final String oldAlias, final Str
             tstore.setCertificateEntry(newAlias, certificate);
             tstore.store(os, passkey.toCharArray());
         } catch (final IOException | NoSuchAlgorithmException | CertificateException | KeyStoreException ex) {
-            IOUtils.closeQuietly(os);
             LOG.error("Unable to update the Certifiate: ", ex.getLocalizedMessage(), ex);
             throw new CertificateManagerException(ex.getMessage(), ex);
+        } finally {
+            IOUtils.closeQuietly(os);
```

Comment:
Unsure if we can make this change. We are returning the actual stream in the method. Are we doing anything with the stream after this method returns, and if we are, would closing it affect the implementation?

Reply:
nice catch.  this method uses by updateCertificate.  We will close stream in updateCertificate instead.